README.md

```diff
@@ -3,17 +3,24 @@
 ![ci](https://github.com/grpc-ecosystem/grpc-health-probe/workflows/ci/badge.svg)
 ![GitHub all releases](https://img.shields.io/github/downloads/grpc-ecosystem/grpc-health-probe/total)
 
-
 The `grpc_health_probe` utility allows you to query health of gRPC services that
 expose service their status through the [gRPC Health Checking Protocol][hc].
 
+`grpc_health_probe` is meant to be used for health checking gRPC applications in
+[Kubernetes][k8s], using the [exec probes][execprobe].
+
+> :warning: [**Kubernetes v1.23 has now introduced built-in gRPC health checking**][k8s-new]
+> capability as an alpha feature. As a result, you might no longer need to use this tool and use the
+> native Kubernetes feature instead.
```

Comment:
@tony-assembly interesting... might be worth looking into (where's bmon when you need him?)

Reply:
i know 😢 how do you propose we give this a k8s feature a crack? I'd be happy to help

Reply:
looks like we might just be able to plug it in! https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe

Reply:
We can try it out in RND... I'll create a ticket

Reply:
🙌